jni/events/basesynthevent.cpp

```diff
@@ -70,6 +70,18 @@ BaseSynthEvent::~BaseSynthEvent()
 
 /* public methods */
 
+void BaseSynthEvent::play()
+{
+    released = false;
+    BaseAudioEvent::play();
+}
+
+void BaseSynthEvent::stop()
+{
+    released = true;
+    BaseAudioEvent::stop();
+}
+
```

Comment:
SynthEvents now maintain a released state. This is reset upon requesting its playback and set upon requesting a halt of its playback.